spec/models/manageiq/providers/ansible_playbook_workflow_spec.rb

```diff
@@ -211,7 +212,7 @@
     it "ansible-runner completed" do
       expect(response_async).to receive(:running?).and_return(false)
 
-      response = Ansible::Runner::Response.new(response_async.dump.merge(:return_code => 0))
+      response = Ansible::Runner::Response.new(**(response_async.dump.merge(:return_code => 0)))
```

Comment:
I think the parens here are redundant.

Reply:
Also, I like

```ruby
response = Ansible::Runner::Response.new(**response_async.dump, :return_code => 0)
```

Reply:
That was pulled out of this PR and is already fixed in master a4388d42582b544d57cb15b3eff50d8fcffcce32 😉